README.md

```diff
@@ -825,6 +794,7 @@ WHERE
 
 Feature | Notes
 :--- | :---
+`UseAzureManagedIdentity` | Since the update of Microsoft.Data.SqlClient in sink release 5.8.0 Active Directory auth capabilities of SqlClient can be used. You can specify one of the supported AD authentication methods, which include Azure Managed Identites, directly in the connection string. Refer to the [SqlClient documentation](https://learn.microsoft.com/en-us/sql/connect/ado-net/sql/azure-active-directory-authentication?view=sql-server-ver16) for details.
```

Comment:
spelling: Identities

Reply:
Thanks a lot to our man from down under. :)